libs/pages/environments/feature/src/lib/components/deployment-rules/create-deployment-rule/create-deployment-rule.tsx

```diff
@@ -0,0 +1,92 @@
+import { BaseLink } from '@console/shared/ui'
+import { AppDispatch, RootState } from '@console/store/data'
+import { fetchClusters, selectClustersEntitiesByOrganizationId } from '@console/domains/organization'
+import { Cluster, ProjectDeploymentRuleRequest, WeekdayEnum } from 'qovery-typescript-axios'
+import { useEffect } from 'react'
+import { useForm } from 'react-hook-form'
+import { useDispatch, useSelector } from 'react-redux'
+import { useNavigate, useParams } from 'react-router'
+import { postDeploymentRules } from '@console/domains/projects'
+import { ENVIRONMENTS_DEPLOYMENT_RULES_URL, ENVIRONMENTS_URL } from '@console/shared/utils'
+import { CreateDeploymentRulePage } from '@console/pages/environments/ui'
+import { Value } from '@console/shared/interfaces'
+
+export function CreateDeploymentRule() {
+  const { organizationId = '', projectId = '' } = useParams()
+  const listHelpfulLinks: BaseLink[] = [{ link: '#', linkLabel: 'How to configure my application', external: true }]
+  const { control, handleSubmit, setValue } = useForm()
+
+  const dispatch = useDispatch<AppDispatch>()
+  const navigate = useNavigate()
+
+  const clusters = useSelector<RootState, Cluster[]>((state) =>
+    selectClustersEntitiesByOrganizationId(state, organizationId)
+  )
+
+  useEffect(() => {
+    dispatch(fetchClusters({ organizationId }))
+
+    const weekdaysSelection = [
+      {
+        label: 'MONDAY',
+        value: 'MONDAY',
+      },
+      {
+        label: 'TUESDAY',
+        value: 'TUESDAY',
+      },
+      {
+        label: 'WEDNESDAY',
+        value: 'WEDNESDAY',
+      },
+      {
+        label: 'THURSDAY',
+        value: 'THURSDAY',
+      },
+      {
+        label: 'FRIDAY',
+        value: 'FRIDAY',
+      },
+    ]
+
+    setValue('timezone', 'UTC')
+    setValue('start_time', '08:00')
+    setValue('stop_time', '19:00')
+    setValue('mode', 'PRODUCTION')
+    setValue('auto_deploy', false)
+    setValue('auto_delete', false)
+    setValue('auto_stop', false)
+    setValue('wildcard', ' ')
+    setValue('weekdays', weekdaysSelection)
+  }, [setValue, dispatch, organizationId])
+
+  const onSubmit = handleSubmit(async (data) => {
+    if (data) {
+      const fields = data as ProjectDeploymentRuleRequest
```

Comment:
should be a `let` because you modify the variable after

Reply:
The variable is modify inside the same function than his declaration so it should be a const here (vscode retrieve an error if I pass it to let)